scene/2d/tile_map.cpp

```diff
@@ -61,6 +61,8 @@ void TileMap::_notification(int p_what) {
 			}
 
 			pending_update = true;
+			_clear_quadrants();
```

Comment:
This line is kind of useless since this function is called first on _recreate_quadrants(), it's not so expensive to call it twise since it'll be empty latter, but maybe we can delete this line for easier readability.

Reply:
https://github.com/authenticate/godot/blob/master/scene/2d/tile_map.cpp#L905